lib/init.coffee

```diff
@@ -10,4 +14,44 @@ module.exports =
         type: 'string'
 
   activate: ->
-    console.log 'activate linter-gjslint'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-gjslint.executablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+  deactivate: ->
+    @subscriptions.dispose()
+  provideLinter: ->
+    helpers = require('atom-linter')
+    provider =
+      name: 'gjslint'
+      grammarScopes: ['source.js', 'source.js.jquery', 'text.html.basic']
+      scope: 'file'
+      lintOnFly: true
+      lint: (editor) =>
+        filePath = editor.getPath()
+        cwd = path.dirname(filePath)
+        tempFile path.basename(filePath), editor.getText(), (tmpFilePath) =>
+          params = [
+            '--nobeep',
+            '--quiet',
+            tmpFilePath
+          ]
+          return helpers.exec(@executablePath, params, {cwd}).then (stdout) ->
+            regex = /^Line\s(\d*)\, E\:(\d*):\s(.*)$/
```

Comment:
Should the `\d*` be `\d+`? It seems that if the message is in that format at all it would have a line number and error code.

(I'm not familiar with gjslint though, so who knows, maybe it does output `Line , E:: stuff` messages :stuck_out_tongue:)